### PR TITLE
feat(table): add prop overloading to column

### PR DESCRIPTION
**Pull request by DanielMSchmidt**

### CODE REVIEW

File: packages/table/components/Column.tsx [hunk 1]
```diff
@@ -14,7 +18,11 @@ export interface ColumnProps {
   /**
    * cellRenderer is the function which is creating the cell contents for this column.
    */
-  cellRenderer: (data: any, width: number) => React.ReactNode;
+  cellRenderer: (
+    data: any,
+    width: number,
+    additionalProps: OverloadedProps
```
Comment (mperrotti) [inline packages/table/components/Column.tsx]: Is there a reason this can't be optional?

Comment (DanielMSchmidt) [reply packages/table/components/Column.tsx]: It should at least be an empty object

Comment (GeorgiSTodorov) [reply packages/table/components/Column.tsx]: This will break existing tables in dcos-ui, then?

**Comment by GeorgiSTodorov** [comment]
A test is failing.
Screenshot from 2019-04-04 10-33-38

File: packages/table/components/Column.tsx (hunk 1 above)
Comment (GeorgiSTodorov) [reply packages/table/components/Column.tsx]: It doesn't, I might have misunderstood.

Comment (DanielMSchmidt) [reply packages/table/components/Column.tsx]: It shouldn't, you can just ommit the argument in the cell renderer :)

**Comment by GeorgiSTodorov** [comment]
@DanielMSchmidt , file conflicts.

**Comment by mperrotti** [comment]
@DanielMSchmidt - could you either update this or close it?

**Pull request closed without merging**